moxy-compiler/src/main/java/moxy/compiler/viewstate/ViewInterfaceProcessor.kt

```diff
@@ -94,22 +95,21 @@ class ViewInterfaceProcessor(
     /**
      * Returns ViewMethod for each suitable enclosed method into this interface (but not superinterface)
      */
-    private fun getEnclosedMethods(viewInterface: TypeElement): Set<ViewMethod> {
+    private fun getEnclosedMethods(viewInterface: TypeElement): List<StockViewMethod> {
         val defaultStrategy = getInterfaceStateStrategyType(viewInterface)
         return viewInterface.enclosedElements
             .filter {
                 // ignore all but non-static methods
                 it.kind == ElementKind.METHOD && !it.isStatic()
             }
             .map { getViewMethod(it as ExecutableElement, viewInterface, defaultStrategy) }
-            .toSet()
     }
 
     private fun getViewMethod(
-        methodElement: ExecutableElement,
-        viewInterface: TypeElement,
-        defaultStrategy: TypeElement?
-    ): ViewMethod {
+            methodElement: ExecutableElement,
+            viewInterface: TypeElement,
+            defaultStrategy: TypeElement?
```

Comment:
For kotlin files, we use the official kotlin style guide. I'll update code style in project soon, but for now you can apply it in Settings -> Editor -> Code Style -> Kotlin -> Set from... -> Kotlin style guide